tests/test_query_params.py

```diff
@@ -476,22 +481,22 @@ def test_boolean(self, simulate_request, client, resource):
 
     def test_boolean_blank(self, simulate_request, client, resource):
         client.app.add_route('/', resource)
-        client.app.req_options.keep_blank_qs_values = True
         simulate_request(client=client, path='/', query_string='blank&blank2=')
 
         req = resource.captured_req
         assert req.get_param('blank') == ''
         assert req.get_param('blank2') == ''
         with pytest.raises(falcon.HTTPInvalidParam):
-            req.get_param_as_bool('blank')
+            req.get_param_as_bool('blank', blank_as_true=False)
 
         with pytest.raises(falcon.HTTPInvalidParam):
-            req.get_param_as_bool('blank2')
-        assert req.get_param_as_bool('blank', blank_as_true=True)
-        assert req.get_param_as_bool('blank3', blank_as_true=True) is None
+            req.get_param_as_bool('blank2', blank_as_true=False)
+        assert req.get_param_as_bool('blank')
```

Comment:
Would you mind adding at least one more test, as I feel like our coverage on the (new) default value is a bit thin; e.g. this would test parsing a param that includes the `=` character:

```py
assert req.get_param_as_bool('blank2')
```

Reply:
(Addressed in my commits)